manager/src/main/java/io/gingersnapproject/hotrod/CommandProcessor.java

```diff
@@ -111,6 +113,26 @@ public void putAll(GingersnapHeader header, Map<byte[], byte[]> entryMap) {
       }
    }
 
+   public void getAll(GingersnapHeader header, Set<byte[]> keys) {
+      try {
+         String cacheName = header.getCacheName();
+         UniJoin.Builder<KeyValuePair<byte[], byte[]>> builder = Uni.join().builder();
```

Comment:
After I thought about it, this needs to be done in a different way because https://github.com/gingersnap-project/cache-manager/pull/18/files#r1049741507 will also need it. This can be fine for a short term solution though, to get functionality working.

Reply:
I added the `putAll` to the `Caches` class.